domain/fqdn.go

```diff
@@ -1,8 +1,25 @@
 package domain
 
+import (
+	"encoding/json"
+
+	model "github.com/owasp-amass/open-asset-model"
+)
+
 // FQDN represents a Fully Qualified Domain Name.
+// FQDN implements Asset interface
 type FQDN struct {
 	// Name is the domain name part of the FQDN.
 	// It should be a valid domain name, such as "example" or "subdomain.example.com".
 	Name string `json:"name"`
 }
+
+// GetAssetType returns the asset type. Implements Asset interface.
```

Comment:
```suggestion
// GetAssetType returns the asset type.
```

Reply:
This comment is accurate and makes understanding the code easier. I want to keep it.

Reply:
Thank you for mentioning it, I understand what you mean, but I think it's not entirely accurate, I would say "_the FQDN structure implements the Asset interface_", not its methods, and maybe it seems verbose to put it in each of them

Reply:
I like the comment as well. I agree it's not entirely accurate, as the receiver method is part of the implementation, but does not "implements" the interface itself. What about `Part of the Asset interface`?